Examples/Profiles/Abstract-profile-example.fsh

```diff
@@ -0,0 +1,42 @@
+// @Name: Abstract Profile Example
+// @Description: Abstract profiles can be used to define common constraints used by multiple child profiles. The term "abstract" means that implementers would not create resources conforming to the abstract profile -- instead, they would target conforming to one of non-abstract child profiles. See "Abstract Profile with RuleSets Example" for an alternate approach to this.
+
+Profile:     ExampleAbstractPatientProfile
+Id:          example-abstract-patient-profile
+Parent:      Patient
+Title:       "Example Abstract Patient Profile"
+Description: "Example of an abstract profile of Patient"
+
+// For more information see http://www.hl7.org/fhir/structuredefinition-definitions.html#StructureDefinition.abstract
+* ^abstract = true
+
+// Require at least one value inside the `name` element
+* name 1..*
+
+// Mark elements as MustSupport
+* name and name.given and name.family MS
+
+* deceased[x] 1..1
+
+
+
+// The profiles below will inherit the constraints from the above parent abstract profile
+
+Profile:     ExampleLivingPatientProfile
+Id:          example-living-patient-profile
+Parent:      ExampleAbstractPatientProfile
+Title:       "Example Living Patient Patient Profile"
+Description: "Example of a profile for living patients inheriting from ExampleAbstractPatientProfile"
+
```

Comment:
Hmm...I can't reproduce this behavior: https://fshschool.org/FSHOnline/#/share/3rwq8Xd

Reply:
In SUSHI, we always initialize certain metadata in profiles -- because not all metadata should be inherited as-is.  The `abstract` property is one of the properties we always initialize to `false` ([code](https://github.com/FHIR/sushi/blob/v2.2.6/src/export/StructureDefinitionExporter.ts#L340)).

We did this based on some guidance on Zulip as well as based on existing behavior from Forge ([zulip conversation](https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/Bad.20links.20on.20Detailed.20Description.20tab/near/186766845)).